osier/__init__.py

```diff
@@ -2,4 +2,14 @@
 from .models.dispatch import *
 from .models.capacity_expansion import *
 from .utils import *
-from .equations import *
\ No newline at end of file
+from .equations import *
+
+import unyt as u
+from unyt import kg
+
+try:
+    u.define_unit("megatonnes", 1e9*kg)
+except RuntimeError as e:
+    print(e)
+    u.unit_registry.default_unit_registry.remove("megatonnes")
+    u.define_unit("megatonnes", 1e9*kg)
```

Comment:
What's this for?

Reply:
This unit doesn't exist by default in `unyt`, so I make sure that it's defined locally because it's used elsewhere in `osier`.